### DISCUSSION TITLE
lex friedman podcast cursor inspiration..

### DISCUSSION THREAD

**Discussion by luxus** [olimorris/codecompanion.nvim#315] [Ideas]
hey did you watched the cursor episode of the podcast?
some things are pretty interesting, using the cursor position and git history etc.

**Comment by S1M0N38** [comment]
I'm gonna take a listen

**Reply by olimorris** [reply]
Good shout, I'll watch it in full. I saw Lex asking DHH what he thought of Cursor on the live stream with the Primeagen so know he's very interested in it.

Will be keen to see what ideas we can implement from them.

I'm working on a @workspace agent that will work through the repository to share relevant context with the LLM.
